sktime/pipeline/_make_pipeline.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Pipeline making utility."""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file).
+
+__author__ = ["fkiraly"]
+
+
+def make_pipeline(*steps):
+    """Approximations of uniform order statistic medians.
```

Comment:
This description doesn't look correct to me

Reply:
yes, fixed.
(I just copied a random function to get started)